### DISCUSSION TITLE
Advanced Markers Announced and in Preview

### DISCUSSION THREAD

**Discussion by onnij** [JustFly1984/react-google-maps-api#3111] [Ideas]
Hey Guys,

I recognize that this was literally just announced so I have no expectations on timeline or anything like that. But, I wanted to bring to everyone's attention in case there are some devs with more available time than me to take a gander, and potentially start making this usable with this package.

https://cloud.google.com/blog/products/maps-platform/announcing-highly-customizable-higher-performance-markers?utm_source=gamma&utm_medium=email&utm_campaign=%2BFY22-Q4-global-Maps-email-dl-AdvancedMarkersLaunch&utm_content=advancedmarkers

https://developers.google.com/maps/documentation/javascript/advanced-markers/overview

Looks to be a lot more extensible than the existing Markers system.

**Comment by JustFly1984** [comment]
Your PR is welcome
